gnosis/gp-v2-services
# Use Sushiswap AMM

Introducing Sushiswap as a valid and usable liquidity source for the solver.

### Test Plan
Existing CI.

1. Run the solver, orderbook and swap-ui locally and place an order selling UNI for SUSHI.
2. Observe that we use uniswap to trade UNI<>ETH and then Sushiswap to trade ETH<>SUSHI

https://etherscan.io/tx/0xdd6220190d91240eb61eda5f07e4b966fca32aa18deb9137011c6d60cb2c7674
